src/schemas/execute.yaml

```diff
@@ -0,0 +1,277 @@
+BlockStateCalls:
+  title: Array of calls to be executed at specific, optional block/state
+  properties:
+    blockOverride:
+      title: Block override
+      $ref: '#/components/schemas/BlockOverride'
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
+    calls:
```

Comment:
No I'd leave `blockOverride` also optional. For a multi-call single-block simulation users shouldn't need to override block fields.

Reply:
I *personally* prefer empty arrays over missing/null arrays.  I find it leads to improved developer ergonomics (no need for conditional branching on presence, you can just map/loop over the array).  Functionally, I think both are fine though.

Reply:
if blockoverride is empty, which block is used? latest I guess?

Reply:
The third parameter is a "block tag" which serves to determine the fallback block fields (as well as base state). Something we have to consider is when there are multiple batches of calls (without block override fields):

- They all inherit the "base block" fields. In this case if user provides 2 batches without fields, it will be equivalent to appending the second batch of calls to the first.
- We magically update the fields of future batches

I'm in favor of 1. I think we should keep this method dumb rather than do something unexpected for users.